config/nightwatch.js

```diff
@@ -47,7 +45,12 @@ module.exports = {
         log_path: './logs/selenium',
         host: '127.0.0.1',
         port: selenium_server_port,
-      }
+      },
+      cli_args: { },
+      test_workers: {
+        enabled: true,
+        workers: 10,
```

Comment:
10 seems excessive... I think "auto" is a better option here.  Worth testing, maybe.